cpu/saml21/periph/spi.c

```diff
@@ -97,6 +97,8 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
         case SPI_SPEED_1MHZ:
             f_baud = 1000000;
             break;
+        case SPI_SPEED_4MHZ:
+            return -1;
```

Comment:
I've got a SPI driver for lm4f120 that I'm using for driving ws2812 leds... And for correctly driving these leds, I need a 4Mhz SPI. Should I try to push my SPI driver before or after the new code lands in master ?